### ISSUE TITLE
bino2.1 with QVR opens 2 windows different sizes on screen, on wrong screen

### ISSUE THREAD

**Issue by 63n** [marlam/qvr#2]
This was posted as a bino issue (#26), but perhaps this is really a libQVR issue. 
link:   https://github.com/marlam/bino/issues/26#issue-2196620092

**Comment by marlam** [comment]
Yes, it's related to libqvr, which relies on Qt window management and configuration mechanisms.
